tmdb/episode.go

```diff
@@ -124,6 +124,13 @@ func (episode *Episode) ToListItem(show *Show, season *Season) *xbmc.ListItem {
 		Art: &xbmc.ListItemArt{},
 	}
 
+	if config.Get().AllowKodiChangeArtworks {
```

Comment:
@antonsoroko Let's remove this setting at all, but, DBID is always pointing to library item. You can see in `infolabels.go`.
So if something is in Kodi library, let's insert that ID into DBID, if not - always insert fake id.

Reply:
hm, in this case if user changes something in kodi library it also will be reflected in elementum. so it will be consistent.
i like this idea. will try to implement and test it.

Reply:
@elgatito 
when i do something like
```
-       if config.Get().AllowKodiChangeArtworks {
+       if lm, err := library.GetMovieByTMDB(movie.ID); lm != nil && err == nil {
+               item.Info.DBID = lm.UIDs.Kodi
+       } else {
fakeDBID := util.GetMovieFakeDBID(movie.ID)
if fakeDBID > 0 {
item.Info.DBID = fakeDBID
```

i got 
`import cycle not allowed: import stack: [github.com/elgatito/elementum/tmdb github.com/elgatito/elementum/library github.com/elgatito/elementum/tmdb] (typecheck)`


i am not sure how to fix this without rewriting a lot of existing code to break the cycle.